tests/test_plugins_os_unix_linux_debian_apt.py

```diff
@@ -14,6 +17,7 @@
     ],
 )
 def test_apt_logs(test_file, target_unix, fs_unix):
+    tz = timezone(timedelta(hours=0))
```

Comment:
Is this not the same as `timezone.utc`?

Reply:
Fixed in [6d1aec6](https://github.com/fox-it/dissect.target/pull/128/commits/6d1aec642fa16c69950ff364c3c97f8ac820b371)